authentication/jwt_session.py

```
@@ -0,0 +1,220 @@
+    Refresh tokens with grace period deduplication and invalidation check.
+
+    Returns:
+        dict with 'access' and 'refresh' token strings
+
+    Raises:
+        InvalidToken if the token is revoked or invalid
+        AuthenticationFailed if user is no longer active
+    """
+    try:
+        refresh = SessionRefreshToken(refresh_token_str, verify=False)
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)